mayastor-test/test_nexus.js

```diff
@@ -29,6 +29,12 @@ const uringFile = '/tmp/uring-backend';
 const diskSize = 64 * 1024 * 1024;
 // external IP address detected by common lib
 const externIp = common.getMyIp();
+// port at which iscsi replicas are available
+const iscsiReplicaPort = '3261';
+
+// NVMEoF frontends don't play nicely with iSCSI backend at the time of writing,
+// so temporarily disable these tests.
+const doIscsiReplica = false;
```

Comment:
we just discussed with @gila  yesterday that we need to move iscsi target to a different process (to the process that contains nvmf target). So it looks that by doing that we could solve this problem as well. No need to change anything in this PR, just a note for myself and Jeffry. 👍